tracer/test/Datadog.Trace.Tests/Configuration/ExporterSettingsTests.cs

```diff
@@ -16,66 +17,191 @@ namespace Datadog.Trace.Tests.Configuration
-            Assert.Equal(expected: TracesTransportType.Default, actual: config.TracesTransport);
-            Assert.Equal(expected: MetricsTransportType.UDP, actual: config.MetricsTransport);
-            Assert.Equal(expected: new Uri($"http://127.0.0.1:8126"), actual: config.AgentUri);
-            Assert.Equal(expected: 8125, actual: config.DogStatsdPort);
-            Assert.False(config.PartialFlushEnabled);
-            Assert.Equal(expected: 500, actual: config.PartialFlushMinSpans);
+            var settings = new ExporterSettings();
+            CheckDefaultValues(settings);
+        }
+
+        [Fact]
+        public void AgentUri()
+        {
+            var param = "http://someUrl";
+            var uri = new Uri(param);
+            var settings = new ExporterSettings { AgentUri = uri };
+            var settingsFromSource = Setup("DD_TRACE_AGENT_URL", param);
+
+            AssertHttpIsConfigured(settingsFromSource, uri);
+            AssertHttpIsConfigured(settings, uri);
+        }
+
+        [Fact]
+        public void UnixAgentUriOnWindows()
+        {
+            var param = @"C:\temp\someval";
+            var uri = new Uri($"unix://{param}");
+
+            var settingsFromSource = Setup("DD_TRACE_AGENT_URL", $"unix://{param}");
+            AssertUdsIsConfigured(settingsFromSource, param.Replace("\\", "/"));
+            settingsFromSource.AgentUri.Should().Be(uri);
         }
 
         [Fact]
-        public void PartialFlushVariables_Populated()
+        public void InvalidAgentUrlShouldNotThrow()
         {
-            var config = Setup(FileExistsMock(), "DD_TRACE_PARTIAL_FLUSH_ENABLED:true", "DD_TRACE_PARTIAL_FLUSH_MIN_SPANS:999");
-            Assert.True(config.PartialFlushEnabled);
-            Assert.Equal(expected: 999, actual: config.PartialFlushMinSpans);
+            var param = "http://Invalid=%Url!!";
+            var settingsFromSource = Setup("DD_TRACE_AGENT_URL", param);
+            CheckDefaultValues(settingsFromSource);
+        }
+
+        [Fact]
+        public void AgentHost()
+        {
+            var param = "SomeHost";
+            var settingsFromSource = Setup("DD_AGENT_HOST", param);
+
+            AssertHttpIsConfigured(settingsFromSource, new Uri("http://SomeHost:8126"));
+        }
+
+        [Fact]
+        public void AgentPort()
+        {
+            var param = 9333;
+            var settingsFromSource = Setup("DD_TRACE_AGENT_PORT", param.ToString());
+
+            AssertHttpIsConfigured(settingsFromSource, new Uri("http://127.0.0.1:9333"));
+        }
+
+        [Fact]
+        public void TracesPipeName()
+        {
+            var param = @"C:\temp\someval";
+            var settings = new ExporterSettings() { TracesPipeName = param };
+            var settingsFromSource = Setup("DD_TRACE_PIPE_NAME", param);
+
+            AssertPipeIsConfigured(settingsFromSource, param);
+            settings.TracesPipeName.Should().Be(param);
```

Comment:
I think this is already covered by `AssertPipeIsConfigured` so it can be removed